src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -134,8 +134,8 @@ def default_api_version(self):
         ResourceType.MGMT_NETWORK: '2020-05-01',
         ResourceType.MGMT_COMPUTE: SDKProfile('2020-06-01', {
             'resource_skus': '2019-04-01',
-            'disks': '2020-05-01',
```

Comment:
is `2020-06-1` api-version available on all regions including sovereign cloud, e.g. Azure China?
@VybavaRamadoss

Reply:
/hold
hold on a moment for confirm, thanks.